### DISCUSSION TITLE
Feature request: Go to file

### DISCUSSION THREAD

**Discussion by connor** [community/community#10890] [Mobile]
Hello!

I'm a frequent user of the _Go to file_ feature on web. It would be lovely if the mobile apps (I am on iOS) supported that!

🙏 thank you for considering!

**Comment by amrehman** [comment]
Hi @connor thanks a lot for getting in touch!

I am happy to tell you that we support "Jump To File" feature on our iOS app. I will try to guide you as best as I can how you can use this feature.
- Open a Pull Request on your choice on iOS and navigate to the files view.
- At the bottom of the screen you will see two sets of buttons.
<img width="196" alt="image" src="https://user-images.githubusercontent.com/17592342/152514618-654d207a-1a5c-4491-8b05-866c93d6c51a.png">
- The button on the right (with the file and arrow icon) is essentially the "Jump to File" button. Once you tap it, you will get a list of files in the pull request and selecting a file there will enable you to relevant area of the pull request you are interested in.

If there is anything else we can help you out with, please do get in touch!
Thanks a lot!

**Comment by connor** [comment]
Hi @amrehman, thanks so much for the thorough walkthrough!

I am aware of the ability to jump to a file on a PR, but am looking for the ability to go to a file on the repo page. Similar to this on web -

![8F1E51C7-2EBC-47F3-9FEF-1EF7312DEC2C](https://user-images.githubusercontent.com/97697/152548745-34c35ea7-893e-4145-946f-87185b9c4169.png)

**Reply by amrehman** [reply]
Thanks for the detailed description!

I now understand what you mean, I will discuss this with our team and internally track this issue.
We will keep a lookout for similar requests and will try to come up with a solution for you in the future!

Thanks a lot for your feedback!
